docs/cn/contributor/Contributor-Getting-Started.md

```diff
@@ -1,227 +1,262 @@
-如果你还未安装`git`，请先进行[安装git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)。
+您需要有一个GitHub帐户才能向Alluxio代码仓库贡献代码。
 
-### GitHub账户
+您需要知道您的GitHub电子邮件地址才能进行贡献。 您可以在个人资料电子邮件设置中找到你的邮件地址。
 
-Alluxio源码托管在GitHub上，仓库地址为[Alluxio repository](https://github.com/Alluxio/alluxio)。
+### 配置您的Git电子邮件
 
-你需要一个GitHub账户来贡献源码，如果还没有，请先[注册一个GitHub账户](https://github.com/)。
+在向Alluxio提交代码之前，您应该确认你的Git电子邮件设置正确。
+[请访问有关设置电子邮件的说明。](https://help.github.com/articles/setting-your-email-in-git/).
 
-另外你还需要知道[你的GitHub账户绑定了哪个邮箱](https://help.github.com/articles/adding-an-email-address-to-your-github-account/)。
+## 分叉Alluxio代码仓库
 
-## Fork Alluxio源代码库
+为了向Alluxio贡献代码，您先需要叉Alluxio代码仓库（repo）。 如果您还没分叉代码仓库，您可以访问[Alluxio代码仓库](https://github.com/Alluxio/alluxio)并按页面右上角的Fork按钮。 之后，您将拥有自己的Alluxio代码仓库分支。
 
-在向Alluxio贡献源码之前，你首先需要fork Alluxio源代码库。如果你还未这么做，先进入[Alluxio仓库](https://github.com/Alluxio/alluxio)，再点击页面右上角的Fork按钮， 之后你便有了Alluxio源代码库的fork了。
 
-在fork Alluxio源代码库后，你需要从该fork创建一个本地的副本，这会将该fork里的文件拷贝到你的本地电脑。使用以下命令创建副本：
+分叉Alluxio代码仓库之后，应创建分叉代码仓库的本地克隆。 这会将您分叉的文件复制到您的计算机上。 您可以使用以下命令克隆您的分叉：
 
 ```console
 $ git clone https://github.com/YOUR-USERNAME/alluxio.git
 $ cd alluxio
 ```
 
-这会将副本拷贝在`alluxio/`目录下。
+这将在alluxio/目录下创建克隆。
 
-为了将远程的Alluxio源码改动更新到你本地的副本中，你需要创建一个指向远程Alluxio源代码库的源。在刚拷贝的副本的目录下，运行：
+要将更改从开源Alluxio原始代码库提取到克隆中，您需要创建一个指向Alluxio原始代码库的新远程代码存储库。 要添加新的远程代码存储库，请在新创建的克隆目录中运行：
 
 ```console
 $ git remote add upstream https://github.com/Alluxio/alluxio.git
 ```
 
-运行以下命令可以查看远程仓库的url：
+这将创建一个称为上游的远程代码仓库指向Alluxio原始代码仓库。 您可以用以下命令查看远程代码仓库的URL: 
 
 ```console
 $ git remote -v
 ```
 
-这会显示`origin`（你的fork）以及`upstream`（Alluxio仓库）的url。
-
-## 配置Git邮箱
-
-在向Alluxio提交commit之前，你需要先确认你的Git邮箱设置正确，请参考该[邮箱设置指南](https://help.github.com/articles/setting-your-email-in-git/)。
+这会显示远程代码仓库的URL，包括您的分叉和上游代码仓库（Alluxio原始代码仓库）。
 
 ## 编译Alluxio
 
-既然你已经有了Alluxio源码的一个本地副本，那现在就可以编译Alluxio啦！
+您现在可以编译Alluxio！
+
+在本地克隆目录中，使用以下命令编译Alluxio：
 
-在本地副本目录下，运行以下命令编译Alluxio：
 
 ```console
-$ mvn clean install
+$ man clean install
 ```
 
-该命令会编译整个Alluxio，并且运行所有测试，这可能会花费几分钟。
+这将编译所有的Alluxio，并运行所有测试。取决于您的硬件，这可能需要几分钟到半小时才能完成。
+
+如果您想重新编译而不运行所有检查和测试，您可以运行：
 
-如果你仅仅只需要重新编译，而不需要运行检查和测试，可以运行：
 
 ```console
-$ mvn -T 2C clean install -DskipTests -Dmaven.javadoc.skip -Dfindbugs.skip -Dcheckstyle.skip -Dlicense.skip
+$ mvn -T 2C clean install -DskipTests -Dmaven.javadoc.skip -Dfindbugs.skip \
+  -Dcheckstyle.skip -Dlicense.skip
 ```
 
-这应该用不到一分钟。
+此编译所花时间应少于1分钟。
 
-这里有更多[编译Alluxio源代码]({{ '/cn/contributor/Building-Alluxio-From-Source.html' | relativize_url }})。
+这里有更多
+[细节编译Alluxio源代码。]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}).
 
-## 领取一个New Contributor任务
+## 领取一个新贡献者任务
 
-Alluxio的ticket中有许多不同的等级，它们分别是：**New Contributor**、**Beginner**、**Intermediate**、 **Advanced**。新的开发者应该在进行更高级的任务之前先完成两个**New Contributor**任务。**New Contributor**任务非常容易，不需要了解过多的关于代码的细节；**Beginner**任务通常只需要修改一个文件；**Intermediate**任务通常需要修改多个文件，但都在同一个包下；**Advanced**任务通常需要修改多个包下的多个文件。
+Alluxio事件表具有多个级别。级别为：**新贡献者**，**初学者**，**中级**，**高级**。
```

Comment:
The previous version sounds better. And we don't need to translate **New Contributor**、**Beginner**、**Intermediate**、 **Advanced** here